src/apple/system.rs

```diff
@@ -55,11 +55,16 @@ pub struct System {
 
 impl Drop for System {
     fn drop(&mut self) {
-        if let Some(conn) = self.connection {
-            unsafe {
-                ffi::IOServiceClose(conn);
+        if let Some(_conn) = self.connection {
+            #[cfg(target_os = "macos")]
```

Comment:
Maybe better to put this `cfg` on the `if let`, no?

Reply:
The reason I put it here is because otherwise an iOS build would warn that the `self.connection` field is unused, resulting in a higher number of `cfg()` blocks elsewhere.

Reply:
If the `connection` field isn't used on iOS, then put a `cfg` on the field too. No need to keep it around on a target if it's not used.

Reply:
NIce, thanks!